sql/src/main/java/org/apache/druid/sql/calcite/rel/Grouping.java

```diff
@@ -92,19 +104,25 @@ private Grouping(
 
   public static Grouping create(
       final List<DimensionExpression> dimensions,
+      final Subtotals subtotals,
       final List<Aggregation> aggregations,
       final DimFilter havingFilter,
       final RowSignature outputRowSignature
   )
   {
-    return new Grouping(dimensions, aggregations, havingFilter, outputRowSignature);
+    return new Grouping(dimensions, subtotals, aggregations, havingFilter, outputRowSignature);
```

Comment:
Should `havingFilter` be annotated with `@Nullable` for this method as well?

Reply:
Yes, it should, I'll add it.